File: wlada-si.22.50.beauty_code-486bf53b528c/Aplikacija/wwwroot/assets/js/termin.js
export class Termin
{
    constructor(id,datum,vremePocetka,vremeKraja,ukupnaCena,radnik,korisnik,salon,listaUsluga)
    {
        this.id = id;
        this.datum = datum;
        this.vremePocetka = vremePocetka;
        this.vremeKraja = vremeKraja;
        this.ukupnaCena = ukupnaCena;
        this.radnik = radnik;
        this.korisnik = korisnik;
        this.salon = salon;

        this.listaUsluga = [];
        this.listaUsluga=listaUsluga;

    }
}

File: wlada-si.22.50.beauty_code-486bf53b528c/Aplikacija/Server/Models/Termin.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Text.Json.Serialization;

namespace Models
{
    public class Termin {
        
        [Key]
        public int ID { get; set; }
        
        [Required]
        public DateTime Datum { get; set; }
        
        [Required]
        public DateTime VremePocetka { get; set; }
        
        [Required]
        public DateTime VremeKraja { get; set; }
        
        [Required]
        public int UkupnaCena { get; set; }
        
        [JsonIgnore]
        public List<Usluga> ListaUsluga { get; set; }
        
        [JsonIgnore]
        public Radnik Radnik { get; set; }
        
        [JsonIgnore]
        public Korisnik Korisnik { get; set; }
        
        [JsonIgnore]
        public Salon Salon { get; set; }
    }
}